src/content/docs/es/guides/bursa/003-commands.md

```
@@ -0,0 +1,78 @@
+
+***
+
+<a name="wallet"></a>
+
+## Usar la línea de comandos para crear una billetera y generar archivos de billetera
+
+Podemos usar la línea de comandos para crear una billetera y generar todos los archivos que necesitaremos para administrar nuestra billetera de Cardano.
+
+Para este ejemplo creamos los archivos de billetera en la carpeta `dev` usando la bandera `--output` y dándole un directorio donde generar los archivos.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)